java/src/main/resources/template.properties

```diff
@@ -161,6 +161,18 @@ gcs.bigquery.output.dataset=
 gcs.bigquery.output.table=
 gcs.bigquery.temp.bucket.name=
 
+#GCStoBigTable properties
+# supported formats avro,parquet,csv
+gcs.bigtable.input.format=<csv|parquet|avro>
+#gcs file location
+gcs.bigtable.input.location=<input file location>
```

Comment:
Remove whatever is after =
These values are used for validation. Line 166 is the only correct usage

Reply:
made the changes and committed